lib/commands/fastboot.js

```diff
@@ -15,27 +16,29 @@ module.exports = {
     { name: 'assets-path', type: String, default: 'dist' }
   ],
 
-  runCommand: function(appName, options) {
-    var commandOptions = this.commandOptions;
-    var outputPath = commandOptions.outputPath;
-    var assetsPath = commandOptions.assetsPath;
+  startServer: function(options) {
     var ui = this.ui;
+    var self = this;
 
-    ui.writeLine("Installing FastBoot npm dependencies");
+    if (this._fastBootListener) {
```

Comment:
The issue would affect `dist/fastboot/node_modules`, which we `require` in  [`buildWhitelistedRequire`](https://github.com/ember-fastboot/fastboot/blob/f1b311d/lib/ember-app.js#L92)